PageViews/PageViews.module.less

```diff
@@ -17,11 +17,12 @@
 	}
 
 	.navButtonContainer {
-		overflow: hidden;
-		z-index: 10;
-		top: ~"calc(50% - " (@sand-button-height + 12) / 2 ~")";
 		position: absolute;
 		width: 100%;
+		height: @sand-button-height * 2;
+		top: ~"calc(50% - " (@sand-button-height * 2) / 2 ~")";
+		z-index: 10;
+		overflow: hidden;
```

Comment:
As we discussed, I think removing `overflow: hidden;` will solve the problem.

Reply:
It's fixed.